webgrid/flask.py

```diff
@@ -50,3 +50,8 @@ def xls_as_response(self, workbook, filename):
         buf.seek(0)
         return send_file(buf, mimetype='application/vnd.ms-excel',
                          as_attachment=True, attachment_filename=filename)
+
+    def csv_as_response(self, csv, filename):
+        csv.seek(0)
+        return send_file(csv, mimetype='application/vnd.ms-excel',
```

Comment:
Any reason not to use `text/csv` for the mimetype?

Reply:
Nope. I will switch it